.github/workflows/haskell-ci.yml

```diff
@@ -60,6 +68,15 @@ on:
           Set this to the value of `runs-on`, *not* `runner.os`.
         type: string
         required: true
+      pre-hook:
```

Comment:
Does it make sense to have a `post-hook` as well? Or can you just put other actions after `haskell-ci` in your workflow file to accomplish this?

Reply:
I just didn't have a use-case for it. Would be easy to add.